ui/src/main/webapp/src/app/registered-application/application-queue-list.component.html

```diff
@@ -6,14 +6,13 @@
                  aria-valuemin="0" aria-valuemax="100" aria-valuenow="100"
                  style="width: 100%">
                 <div class="file-info">
-                    <span>{{application.title}}</span>
+                    <span>{{application.title}} 100%</span>
```

Comment:
I think it is another reason to reevaluate our progress bar design. I will look at OpenShift and see what they use...